core/templates/pages/topic-viewer-page/info-tab/topic-info-tab.directive.ts

```diff
@@ -45,7 +45,8 @@ angular.module('oppia').directive('topicInfoTab', ['UrlInterpolationService',
         '/pages/topic-viewer-page/info-tab/topic-info-tab.directive.html'),
       controllerAs: '$ctrl',
       controller: [
-        'WindowDimensionsService', function(WindowDimensionsService) {
+        '$rootScope', 'WindowDimensionsService', function(
```

Comment:
Maybe bring function down to the next line too?

Reply:
Done